util/database.ts

```diff
@@ -0,0 +1,14 @@
+import { DataSource } from "typeorm"
+import {Movie} from "../model/Movie"
+
+export const AppSource = new DataSource({
+    type: "postgres",
+    host: "localhost",
+    port: 5432,
+    username: "postgres",
+    password: "martina",
+    database: "moviesDb",
```

Comment:
We should not store any credentials as part of the source code, for now, it is okay since this is just an example, but on real projects, there are other services where this data is stored. Maybe you can keep these values as variables in a .env file.